src/libclipper/include/clipper/containers.hpp

```diff
@@ -12,6 +12,8 @@
 
 namespace clipper {
 
+using Deadline = std::chrono::time_point<std::chrono::high_resolution_clock>;
```

Comment:
I think we should do this in a separate PR, as it involves changes in several pieces of infrastructure that aren't immediately related to this task.